doc/modules/classes.rst

```diff
@@ -1391,6 +1389,18 @@ Recently deprecated
 ===================
 
 
+To be removed in 0.21
```

Comment:
nitpickiest nitpick: shouldn't that go below removed in 0.20? though no strong opinion.